heudiconv/convert.py

```diff
@@ -125,17 +126,15 @@ def prep_conversion(sid, dicoms, outdir, heuristic, converter, anon_sid,
     #  1. add a test
     #  2. possibly extract into a dedicated function for easier logic flow here
     #     and a dedicated unittest
-    if not reuse_conversion_table and \
-        op.exists(target_heuristic_filename) and \
-        file_md5sum(target_heuristic_filename) != file_md5sum(heuristic.filename):
+    if (op.exists(target_heuristic_filename) and
+        file_md5sum(target_heuristic_filename) != file_md5sum(heuristic.filename)):
+        # remake conversion table
```

Comment:
@yarikoptic at present if the edit file exists we would skip this check.

Reply:
@yarikoptic ping - the more I think about this, the more I like the idea of regenerating the conversion table if there has been a change to the heuristic used.

Reply:
I think we did have that intention originally but literally introduced a bug with the initial check (should have been `if  reuse_conversion_table`)... so let it be indeed